src/SB/Game/zNPCTypeKingJelly.cpp

```diff
@@ -18,6 +57,98 @@ void lightning_ring::create()
     //store 0 into 0x7c
 }
 
+void foo()
+{
+    xAnimTable* foo = xAnimTableNew("FOO", 0, 0);
+}
```

Comment:
I didn't mean for you to add this, it was just for illustrative purposes, we don't want to commit garbage functions just for the sake of <1% of a function match